### DISCUSSION TITLE
Handling Maximum Simultaneous Jobs

### DISCUSSION THREAD

**Discussion by bcook75** [RandomEngy/VidCoder#881] [Ideas]
I feel like a kid who has just discovered a new toy (after having used VidCode, both release and beta, for several years - yeah, I'm slow). I just figured out how to use the "Maximum Simultaneous Encoding Jobs" setting to my advantage. At least partly. For a long time I have used the release version and the beta version at the same time, each one handling one DVD drive, until I discovered some as-yet-unanswered maybe issues with the beta. Then I discovered that I could use one instance of the release version to handle both DVD drives at the same time! Wonderful. However, when trying to encode several things from ONE DVD, I think the whole "simultaneous" deal might be causing more congestion than it is helping. 

I'm open to being shown I'm wrong, but it would seem to make more sense TO ME to limit the number of simultaneous jobs FROM A SINGLE DRIVE to just 1, regardless of the setting of this option. That way I could have several jobs queued for each of 2 drives, and have one job from each drive running in parallel, but not be sending either drive crazy trying to retrieve multiple tracks at the same time.

**Comment by RandomEngy** [comment]
That's a really good idea. I needed to add some limits on simultaneous jobs for other hardware encoders and this will fit right in with that work.

**Comment by RandomEngy** [comment]
As of the newly released v6.38 Beta it restricts to just one encode from a DVD drive at a time.

For now it won't add on a second simultaneous encode if the next item uses a drive that's already encoding something. But you could alternate the jobs between drives to allow both to run at once and avoid simultaneous jobs on the same drive.
